test/test_tree_2d_euler.jl

```diff
@@ -26,6 +26,14 @@ EXAMPLES_DIR = pkgdir(Trixi, "examples", "tree_2d_dgsem")
       linf = [0.006614198043413566, 0.0006614198043973507, 0.001322839608837334, 0.000165354951256802],
       tspan = (0.0, 0.5))
   end
+  
+  @trixi_testset "elixir_euler_density_wave_restart.jl" begin
+    trixi_include(@__MODULE__, joinpath(EXAMPLES_DIR, "elixir_euler_density_wave_extended.jl"))
+    l2, linf = analysis_callback(sol)
+    eval(:(@test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_density_wave_restart.jl"),
+      # Expected errors are exactly the same as in the elixir_euler_density_wave_extended.jl
+      l2 = $l2, linf = $linf)))
```

Comment:
Why do you need the call to `eval`?

Reply:
Thank you for a review!
`eval` is used to evaluate macros with runtime arguments, I wanted to prove in runtime, that pause doesn't make any difference in result.

Reply:
Are you sure that this is really necessary? I am not convinced, but maybe I am missing something... we did not need this anywhere before, did we?

Reply:
You're rigth. As I understand macros are normally working only with compilation time known arguments. Before we used `@test_trixi_include` with compilation known `linf` and `l2`, but now I try to use `@test_trixi_include` with only in runtime known linf` and `l2`. It throws an error, if I don't apply any modifications on `@test_trixi_include` . You're rigth, that `eval` is not necessary here, but necessary to use quoting `:` in way `:@test_trixi_include`. I'm not quite sure yet, why `:` helps. But it works somehow...